codegen_glibmm/templates/proxy.cpp.templ

```diff
@@ -102,14 +102,21 @@ std::tuple<{{ method.out_args|join(', ', attribute='cpptype_out') }}>
 {% endfor %}
 {% for prop in interface.properties %}
 {% if prop.readable %}
-{{ prop.cpptype_out }} {{ class_name_with_namespace }}::{{ prop.name }}_get()
+{{ prop.cpptype_out }} {{ class_name_with_namespace }}::{{ prop.name }}_get(bool *ok)
 {
     std::vector<Glib::ustring> props = m_proxy->get_cached_property_names();
     Glib::Variant<{{ prop.variant_type }}> b;
     if (std::find(props.begin(), props.end(), "{{ prop.name }}") != props.end()) {
         m_proxy->get_cached_property(b, "{{ prop.name }}");
+        if (ok) {
+            *ok = true;
+        }
     } else {
-        g_print("Todo: lookup value\n");
+        if (ok) {
+            *ok = false;
+        } else {
+            g_warning("Unhandled error while getting property {{ prop.name }}\n");
```

Comment:
Does it perhaps make sense to mention "not in cache" or "property not cached" or something in the error message?

Reply:
I was thinking about this too, but hesitated if it was too much details just related to the implementation. But I will happily rephrase the warning if you think it's better, for me it was basically  a coin toss of whether to include it or not.

Reply:
Probably does not make it any more clear to mention "cache". Caller can always trace it back to this location to find out what is wrong and find out about the "bool *" argument. Perhaps it should be documented, but other details or not documented anywhere so... :)

Looks OK otherwise, I think. Will let @mardy and/or @kursatkobya have a look as well. :)